perfkitbenchmarker/test_util.py

```diff
@@ -66,3 +69,23 @@ def assertSampleListsEqualUpToTimestamp(self, a, b, msg=None):
         ex.message = ex.message + (' (was item %s in list)' % i)
         ex.args = (ex.message,)
         raise ex
+
+
+def testDiskMounts(benchmark_config, mount_point):
+  spec = benchmark_spec.BenchmarkSpec(benchmark_config, 'uid')
```

Comment:
Done. Also renamed the function to `assertDiskMounts` to be more clear about what it does.